localization/english/wc_mercenaries_l_english.yml

```diff
@@ -1,4 +1,130 @@
 ﻿l_english:
+# Ancient
+# Protector
+ mercenary_company_keepers_of_the_grove_article:0 "the "
+ mercenary_company_keepers_of_the_grove:0 "Keepers of the Grove"
+ mercenary_company_ironbarks_article:0 "the "
+ mercenary_company_ironbarks:0 "Ironbarks of [TITLE.GetNameNoTierNoTooltip]"
+# War
+ mercenary_company_stomping_roots_article:0 "the "
+ mercenary_company_stomping_roots:0 "Stomping Roots of [TITLE.GetNameNoTierNoTooltip]"
+# mercenary_company_ironbranch_guard_article:0 "the "
+ mercenary_company_ironbranch_guard:0 "Ironbranch's Guard"
+# Lore
+ mercenary_company_wisdom_forest_article:0 "the "
+ mercenary_company_wisdom_forest:0 "Wisdom of the Forest"
+ mercenary_company_elderleaves_article:0 "the "
+ mercenary_company_elderleaves:0 "Elderleaves of [TITLE.GetNameNoTierNoTooltip]"
+# Wind
+ mercenary_company_woodenbeaks_article:0 "the "
+ mercenary_company_woodenbeaks:0 "Woodenbeaks of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_rootclaws_article:0 "the "
+ mercenary_company_rootclaws:0 "Rootclaws of [TITLE.GetNameNoTierNoTooltip]"
+
+# Aqir
+# Qiraji
+# mercenary_company_cthun_guard_article:0 "the "
+ mercenary_company_cthun_guard:0 "C'thun's Guard"
```

Comment:
This assumes the qiraji worship C'Thun. They may not. Mercenaries are a cultural thing.